docs/guides/app-distribution.md

```diff
@@ -51,14 +51,14 @@ apiApp.command("/hi", (req, ctx) -> {
 
 // OAuth Flow Handler App
 //  expected env variables:
-//   SLACK_APP_CLIENT_ID, SLACK_APP_CLIENT_SECRET, SLACK_APP_REDIRECT_URI, SLACK_APP_SCOPE,
-//   SLACK_APP_OAUTH_START_PATH, SLACK_APP_OAUTH_CALLBACK_PATH
-//   SLACK_APP_OAUTH_COMPLETION_URL, SLACK_APP_OAUTH_CANCELLATION_URL
+//   SLACK_CLIENT_ID, SLACK_CLIENT_SECRET, SLACK_REDIRECT_URI, SLACK_SCOPES,
```

Comment:
The old env variables still work but we start recommending these ones that are compatible with other Bolt frameworks.